### ISSUE TITLE
In scan mode, focus is not in sequential order after expanding any drop down and navigating (In backward direction) 

### ISSUE THREAD

**Issue by SyedRehmanRazvi** [OfficeDev/office-ui-fabric-react#10578]
User Experience: 
Users who rely on screen readers are getting affected as focus is not in sequential order after expanding any drop down and navigating (In backward direction) 

Test Environment: 
OS: Win 10 
OS Version: 1903 (OS Build 18323.1000) 
Browser: Edge
URL: https://developer.microsoft.com/en-us/fabric#/controls/web/dropdown
Screen Reader: Narrator

Repro Steps: 
Step 1: Start Narrator
Step 2: Open above URL using EDGE browser 
Step 3: Now navigate to any drop down and activate using scan mode
Step 4: After expanding drop down, use "Up arrow key" to navigate.
Step 5: Verify focus is in sequential order after expanding any drop down and navigating (In backward direction)

Actual: 
In scan mode, focus is not in sequential order after expanding any drop down and navigating (In backward direction)

Expected: 
In scan mode, focus should be in sequential order after expanding any drop down and navigating (In backward direction)

![image](https://user-images.githubusercontent.com/53210862/65513647-40638e00-def9-11e9-80db-59ea1dd18d9d.png)

**Comment by dzearing** [comment]
@cliffkoh could you help look at this as you're in the dropdown accessibility stuff atm?

**Comment by cliffkoh** [comment]
@dzearing no problem. I am out of office today but will pick this up when I am back.

**Comment by cliffkoh** [comment]
Hi @SyedRehmanRazvi, can you please help me understand what exactly are the steps you perform (perhaps capturing these in a GIF) so I can understand how/where things went wrong? I have tried reproing and I am struggling a bit to understand the problem.

When I expand a Dropdown and press up, I see the Dropdown going to the previous selected option (in scan mode). Please see attached GIF.

![dropdown scan mode](https://user-images.githubusercontent.com/1003246/65788476-028c8280-e110-11e9-981c-4a068ce31d11.gif)

Please record a GIF of what you're doing, and please try to illustrate (perhaps annotating a screenshot) what is it you expect.
